package.json

```diff
@@ -1,33 +1,127 @@
+    "babel-plugin-dev-expression": "^0.2.1",
+    "babel-plugin-syntax-decorators": "^6.13.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-typecheck": "^3.9.0",
+    "babel-plugin-webpack-loaders": "^0.7.1",
+    "babel-polyfill": "^6.13.0",
+    "babel-preset-es2015": "^6.14.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-react-optimize": "^1.0.1",
+    "babel-preset-stage-0": "^6.5.0",
+    "babel-register": "^6.14.0",
+    "classnames": "^2.2.5",
+    "concurrently": "^2.2.0",
+    "copy-webpack-plugin": "^4.0.1",
+    "cross-env": "^2.0.1",
+    "css-loader": "^0.24.0",
+    "del": "^2.2.2",
+    "devtron": "^1.3.0",
+    "electron": "^1.7.2",
+    "electron-debug": "^1.0.1",
+    "electron-builder": "^19.19.1",
+    "electron-devtools-installer": "^2.0.1",
+    "electron-packager": "^8.7.2",
+    "eslint": "^3.4.0",
+    "eslint-config-airbnb": "^10.0.1",
+    "eslint-import-resolver-webpack": "^0.5.1",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^2.2.0",
+    "eslint-plugin-react": "^6.10.3",
+    "express": "^4.14.0",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "node-libs-browser": "^1.0.0",
+    "postcss": "^5.1.2",
+    "postcss-cssnext": "^3.0.0",
+    "postcss-extend": "^1.0.5",
+    "postcss-import": "^10.0.0",
+    "postcss-loader": "^2.0.6",
+    "postcss-nested": "^2.0.2",
+    "postcss-reporter": "^4.0.0",
+    "postcss-simple-vars": "^4.0.0",
+    "react-addons-test-utils": "^15.3.1",
+    "spectron": "^3.3.0",
+    "style-loader": "^0.13.1",
+    "svg-inline-loader": "^0.8.0",
+    "webpack": "^1.13.2",
+    "webpack-dev-middleware": "^1.6.1",
+    "webpack-hot-middleware": "^2.12.2",
+    "webpack-merge": "^0.14.1"
+  },
   "build": {
+    "extraResources": "../dist",
     "appId": "org.livepeer.desktop",
+    "directories": {
+      "output": "release"
+    },
+    "asarUnpack": [
+      "**/app/node_modules/ffmpeg-static/*",
+      "**/app/node_modules/livepeer-static/*"
+    ],
+    "win": {
+      "target": [
+        "nsis"
+      ],
+      "icon": "resources/windows/icon.ico"
+    },
+    "nsis": {
+      "oneClick": true,
+      "installerHeaderIcon": "resources/windows/setup-icon.ico"
+    },
+    "mac": {
+      "icon": "resources/osx/icon.icns",
+      "category": "your.app.category.type"
+    },
     "dmg": {
-      "contents": [
-        {
-          "x": 110,
-          "y": 150
-        },
-        {
-          "x": 240,
-          "y": 150,
-          "type": "link",
-          "path": "/Applications"
-        }
-      ]
+      "icon": "resources/osx/dmg-icon.icns"
     }
   },
-  "devDependencies": {
-    "electron-builder": "^17.1.1",
-    "electron-prebuilt": "~1.3.15"
+  "dependencies": {
+    "app-root-dir": "^1.0.2",
+    "electron-log": "^2.2.7",
+    "ffmpeg-static": "^2.0.0",
+    "livepeer-static": "https://github.com/buildog/livepeer-static.git",
```

Comment:
I'm going through the endpoints to update them.  Will push a PR to your repo when I'm done.